src/index.js

```diff
@@ -98,12 +122,26 @@ export default class ReactUserTour extends Component {
 			currentTourStep.selector,
 			this.props.style.width,
 			this.props.style.height,
-			currentTourStep.position
+			currentTourStep.position,
+			currentTourStep.margin,
+			currentTourStep.horizontalOffset,
+			currentTourStep.verticalOffset
 		);
 		const style = {...this.props.style};
+		const useCustomArrow = (customArrow) =>
```

Comment:
can this function be defined outside of render, better for performance if we don't have to define it every time we call render.